project/Build.scala

```diff
@@ -131,62 +178,82 @@ object DottyBuild extends Build {
     baseDirectory in (Test,run) := (baseDirectory in dotty).value,
 
 
-    libraryDependencies ++= Seq("com.storm-enroute" %% "scalameter" % "0.6" % Test,
+    libraryDependencies ++= Seq(
+      "com.storm-enroute" %% "scalameter" % "0.6" % Test,
       "com.novocode" % "junit-interface" % "0.11"),
     testFrameworks += new TestFramework("org.scalameter.ScalaMeterFramework"),
 
-    // scalac options
     scalacOptions in Global ++= Seq("-feature", "-deprecation", "-language:_"),
-
     javacOptions ++= Seq("-Xlint:unchecked", "-Xlint:deprecation"),
 
     fork in Test := true,
     parallelExecution in Test := false,
 
-    // http://grokbase.com/t/gg/simple-build-tool/135ke5y90p/sbt-setting-jvm-boot-paramaters-for-scala
-    javaOptions <++= (dependencyClasspath in Runtime, packageBin in Compile) map { (attList, bin) =>
-      // put the Scala {library, reflect, compiler} in the classpath
-      val path = for {
-        file <- attList.map(_.data)
-        path = file.getAbsolutePath
-        prefix = if (path.endsWith(".jar")) "p" else "a"
-      } yield "-Xbootclasspath/" + prefix + ":" + path
-      // dotty itself needs to be in the bootclasspath
-      val fullpath = ("-Xbootclasspath/a:" + bin) :: path.toList
-      // System.err.println("BOOTPATH: " + fullpath)
-
-      val travis_build = // propagate if this is a travis build
-        if (sys.props.isDefinedAt(JENKINS_BUILD))
-          List(s"-D$JENKINS_BUILD=${sys.props(JENKINS_BUILD)}")
-        else
-          List()
-      val res = agentOptions ::: travis_build ::: fullpath
-      println("Running with javaOptions: " + res)
-      res
-    }
+    // http://grokbase.com/t/gg/simple-build-tool/135ke5y90p/\
```

Comment:
I'm not sure if breaking URLs to multiple lines is a good idea. This breaks IDE support for them and disallows to simply copy them into browser.